changelog.d/20240319_140241_andrew.bonnell_add_config_user_filter.md

```diff
@@ -0,0 +1 @@
+- [Feature] Introduces the CONFIG_USER Filter. Used to declare unique key:value pairs in config.yml that will be overwritten when running tutor config save. Useful for injecting secrets from an external secrets manager into edx, or other values that may change over time that you need to programmatically fetch. (by @abonnell)
```

Comment:
nit: Useful for injecting secrets from an external secrets manager into edx, or other values that may change over time that you need to programmatically fetch --> The filter can be used for injecting secrets from an external secrets manager into Open edX deployment, or other values that may change over time and need to be fetched programmatically